Owlet Component Documentation

oblogic7 opened the pull request:
**Description:**
Add documentation for Owlet component

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#20258

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

oblogic7 committed:
Owlet component documentation

oblogic7 committed:
Owlet component documentation

rohankapoorcom commented on source/_components/binary_sensor.owlet.markdown:
```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: "Owlet"
```
Title should be different for each platform and the component.

rohankapoorcom commented on source/_components/owlet.markdown:
```diff
@@ -0,0 +1,60 @@
+ha_qa_scale:
+---
+
+The Point component is the main component to integrate the [Minut Point](https://minut.se/). To connect Point, you will have to [sign up for a developer account](https://minut.com/community/developers/) and get a `client_id` and `client_secret` using the `callback url`  as `base_url` + `/api/minut` eg. `http://localhost:8123/api/minut`. The `client_id` and `client_secret` should be used as below.
```
This and the next paragraph seem leftover from a copy/paste.

oblogic7 committed:
Rmove leftover copy pasta

oblogic7 committed:
Rmove leftover copy pasta

rohankapoorcom reviewed (commented):
The logo is not working. See: https://deploy-preview-8217--home-assistant-docs.netlify.com/components/owlet/


I think you have to use a png, not an svg.

oblogic7 committed:
Correct logo file location and case

oblogic7 committed:
Correct logo file location and case

oblogic7 commented:
Looks like I didn't get the file into the supported_brands folder when I dropped it onto the project.  I've updated the file location.  I think SVG will work as there are others in the supported_brands folder.

rohankapoorcom reviewed (dismissed):
Looks good!

frenck commented on source/_components/binary_sensor.owlet.markdown:
```diff
@@ -0,0 +1,38 @@
+<p class='warning'>
+The intended purpose of this component is to enable data logging and automations
+such as battery status updates and charging reminders.  This component should not
+replace the Owlet app nor should it be used for for life critical notifications.
```
```suggestion
replace the Owlet app nor should it be used for life-critical notifications.
```

frenck commented on source/_components/binary_sensor.owlet.markdown:
```diff
@@ -0,0 +1,38 @@
+featured: true
+ha_release: "0.87"
+ha_iot_class: "Cloud Polling"
+ha_qa_scale:
```
Please remove this, since there is no ha_qa_scale.

frenck commented on source/_components/owlet.markdown:
```diff
@@ -0,0 +1,56 @@
+owlet:
+  username: OWLET_USER
+  password: OWLET_PASSWORD
+  name: CUSTOM_NAME
```
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

frenck commented on source/_components/owlet.markdown:
```diff
@@ -0,0 +1,56 @@
+featured: true
+ha_release: "0.87"
+ha_iot_class: "Cloud Polling"
+ha_qa_scale:
```
Please remove the ha_qa_scale.

frenck commented on source/_components/owlet.markdown:
```diff
@@ -0,0 +1,56 @@
+<p class='warning'>
+The intended purpose of this component is to enable data logging and automations
+such as battery status updates and charging reminders.  This component should not
+replace the Owlet app nor should it be used for for life critical notifications.
```
```suggestion
replace the Owlet app nor should it be used for life-critical notifications.
```

frenck commented on source/_components/sensor.owlet.markdown:
```diff
@@ -0,0 +1,39 @@
+featured: true
+ha_release: "0.87"
+ha_iot_class: "Cloud Polling"
+ha_qa_scale:
```
Please remove the ha_qa_scale

frenck commented on source/_components/sensor.owlet.markdown:
```diff
@@ -0,0 +1,39 @@
+<p class='warning'>
+The intended purpose of this component is to enable data logging and automations
+such as battery status updates and charging reminders.  This component should not
+replace the Owlet app nor should it be used for for life critical notifications.
```
```suggestion
replace the Owlet app nor should it be used for life-critical notifications.
```

frenck reviewed (changes requested):
Thanks for the PR @oblogic7, I've left some comments, could you please take a look? 👍

oblogic7 committed:
Requested changes.

Wording.  Remove ha_qa_scale.

oblogic7 committed:
Requested changes.

Wording.  Remove ha_qa_scale.

oblogic7 commented:
@frenck Requested changes are complete.

frenck committed:
:pencil2: Tweak

frenck reviewed (dismissed):
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

MartinHjelmare commented on source/_components/binary_sensor.owlet.markdown:
```diff
@@ -0,0 +1,37 @@
+  You must have the [Owlet component](/components/owlet/) configured to use this sensor.
+</p>
+
+## {% linkable_title Configuration %}
```
Remove this section. It's not correct. The platform is only loaded via discovery from the component.

MartinHjelmare commented on source/_components/binary_sensor.owlet.markdown:
```diff
@@ -0,0 +1,37 @@
+featured: true
+ha_release: "0.87"
+ha_iot_class: "Cloud Polling"
+---
```
We can merge this page with the component page as platforms don't require any configuration.

MartinHjelmare commented on source/_components/owlet.markdown:
```diff
@@ -0,0 +1,54 @@
+logo: owlet.svg
+ha_category: Health
+featured: true
+ha_release: "0.87"
```
0.89

MartinHjelmare commented on source/_components/owlet.markdown:
```diff
@@ -0,0 +1,54 @@
+footer: true
+logo: owlet.svg
+ha_category: Health
+featured: true
```
Remove this.

MartinHjelmare commented on source/_components/sensor.owlet.markdown:
```diff
@@ -0,0 +1,38 @@
+featured: true
+ha_release: "0.87"
+ha_iot_class: "Cloud Polling"
+---
```
We can remove this page and merge content with component page.

oblogic7 committed:
Requested changes

oblogic7 committed:
Merge branch 'owlet' of github.com:oblogic7/home-assistant.github.io into owlet

Conflicts:
	source/_components/owlet.markdown

oblogic7 committed:
Merge branch 'next' of github.com:home-assistant/home-assistant.github.io into owlet

oblogic7 commented:
@MartinHjelmare Something blew up this branch during rebase too.  I'm thinking something is wonky with my PyCharm git integration.  Created a new PR (#8608) from clean working branch.
